packages/react-components/react-menu/src/components/MenuItemLink/MenuItemLink.types.ts

```diff
@@ -0,0 +1,16 @@
+import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+import { MenuItemProps, MenuItemSlots } from '../MenuItem/MenuItem.types';
+
+export type MenuItemLinkSlots = {
+  root: Slot<'a'>;
+} & Pick<MenuItemSlots, 'icon' | 'content' | 'secondaryContent' | 'checkmark'>;
+
+/**
+ * MenuItemLink Props
+ */
+export type MenuItemLinkProps = ComponentProps<MenuItemLinkSlots> & Pick<MenuItemProps, 'disabled'> & { href: string };
```

Comment:
```suggestion
export type MenuItemLinkProps = ComponentProps<MenuItemLinkSlots> & Pick<MenuItemProps, 'disabled'> & { href?: string };
```

AFAIR `href` is optional on `a`

Reply:
yes, but I chose to make it required here, because we want users to leverage `href`, if not they can use a normal `MenuItem`

Reply:
> What about cases with router components i.e. `react-router` or `next/link`? They have `to` as I remember.

composition API? 👀

Reply:
`react-router` and `next/link` actually render anchor tags... the only way I could see that working with our components is to use recomposition and at that point you could simply map `href` to `to`

WDYT?